mitodl/bootcamp-ecommerce
# Update hubspot contact sync code

#### What are the relevant tickets?
Closes #402

#### What's this PR do?
- Removes the `HubspotContactSerializer` which had smapply-specific code
- Uses `profiles.UserProfileSerializer` for syncing contacts instead (like xpro)
- Updates `configure_hubspot_bridge` management command with new fields

#### How should this be manually tested?
- Run `manage.py configure_hubspot_bridge`
- Go to `/profile/edit` (after logging in) and edit your profile.  Enter something for all fields.
- Log in to hubspot and search contacts for the above user.  The changes should be reflected there.
